src/BaseModel.php

```diff
@@ -23,6 +23,7 @@
 use SlaxWeb\Database\Exception\QueryException;
 use SlaxWeb\Database\Interfaces\Library as Database;
 use SlaxWeb\Database\Interfaces\Result as ResultInterface;
+use SlaxWeb\Hooks\Container as Container;
```

Comment:
*\\Container as Container* makes no sense, change to *as HooksContainer* andorder it based on line length.